test/main.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
What are these test packages for? Are these necessary?

Reply:
this were for doing some test, they were deleted